frontmacs-modeline.el

```diff
@@ -0,0 +1,23 @@
+;;; frontmacs-modeline.el --- Setup our modeline.
+;;;
+;;; Commentary:
+;;;
+;;; Code:
+
+(require 'diminish)
+
+;; diminish some modes
```

Comment:
Emacs itself can be frightening, and the eco system surrounding it can be even more intimidating to somebody who might come to read this code afterwards who lacks the context that we have. 

Let's explain the problem, what diminish is, where it can be found, and what we're using it for. For example:

> ;; Out of the box, Every single major and minor mode gets put onto the 
> ;; modeline which clutters it up terribly. Using `diminish` we can hide or
> ;; abbreviate modes that don't contribute meaningfully to the high-level 
> ;; context.
> ;; see https://github.com/myrjola/diminish.el